pkg/osquery/table/program_icons_windows.go

```diff
@@ -90,6 +104,13 @@ func generateUninstallerProgramIcons(ctx context.Context, includeIcon bool) []ma
 				continue
 			}
 
+			// If we're filtering on program names, check for that
+			if len(programNames) > 0 {
```

Comment:
I bet we always want to specify a where clause, so we could return nothing and log an error if this is empty. (Or not bother, approved as is)

Reply:
No logger access anywhere here at the moment -- probably not worth bothering about yet?